mlflow/tracking/artifact_utils.py

```diff
@@ -57,10 +53,8 @@ def _download_artifact_from_uri(artifact_uri, output_path=None):
     :param output_path: The local filesystem path to which to download the artifact. If unspecified,
                         a local output path will be created.
     """
-    artifact_path_module = \
-        get_artifact_repository(artifact_uri).get_path_module()
-    artifact_src_dir = artifact_path_module.dirname(artifact_uri)
-    artifact_src_relative_path = artifact_path_module.basename(artifact_uri)
+    artifact_src_dir = posixpath.dirname(artifact_uri)
```

Comment:
this would be a problem if artifact_uri is a file path on windows right? this function directly powers functions like `load_model` which i assumed would take in windows paths.

Reply:
Yes but I don't think we can make this work with windows paths, at least not without some sacrifices - i.e. what if the artifact name contains '\'? 

I think it's ok not to accept windows paths here because the argument is artifact path which we say is always posix. User can still supply posix path on windows or fully qualified uri.